### PR TITLE
OUI-16

### LINKED ISSUE #16: update dependencies

### PR DESCRIPTION
**Pull request by amgoodfellow**
This branch updates front and back-end dependencies. 

Closes #16

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,5 @@
-#Fri Apr 20 10:34:16 EDT 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.2.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-bin.zip
```
Comment (JoshBrudnak) [inline gradle/wrapper/gradle-wrapper.properties]: Why not update to 5.1.1

File: src/main/react/src/index.js [hunk 2]
```diff
@@ -1,87 +1,49 @@
...
   }
 
   componentDidMount() {
-    adminCheck(token).then(isAdmin => {
-      this.setState({ isAdmin })
-    })
-    isListEmpty().then(isEmpty => {
-      this.setState({ isEmpty })
-    })
+    //adminCheck(token).then(isAdmin => {
```
Comment (JoshBrudnak) [inline src/main/react/src/index.js]: Do you need to keep this commented out code?

**Review by JoshBrudnak — APPROVED** [review]
Other than a few things LGTM. It would also be helpful if this project had a readme.

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (amgoodfellow) [reply gradle/wrapper/gradle-wrapper.properties]: Good point - I've updated it

File: src/main/react/src/index.js (hunk 2 above)
Comment (amgoodfellow) [reply src/main/react/src/index.js]: Actually, I should probably un-comment it lol

**Pull request merged**